config.py

```diff
@@ -1,6 +1,7 @@
 import os
 from datetime import timedelta
 
+GOOGLE_AUTH_CLIENT_ID = "992237180107-lsibe891591qcubpbd8qom4fts74i5in.apps.googleusercontent.com"
```

Comment:
You are not supposed to commit this, instead keep it configurable using an environment variable.

Reply:
Please also make sure you revoke the oAuth application associated with this, if you don't want misuse.

Reply:
Fixed. Thanks!

Reply:
> You are not supposed to commit this, instead keep it configurable using an environment variable.

I actually thought about that, but then ... not sure why I forgot to review it here. Thank you so much @SanketDG for your review!